referencia/patpasscomercio/README.md

```diff
@@ -0,0 +1,385 @@
+# Patpass Rest
+## Patpass Comercio
+### Ambientes y credenciales
+La API REST de Patpass está protegida para garantizar que solamente comercios autorizados por Transbank hagan uso de las operaciones disponibles. La seguridad esta implementada mediante los siguientes mecanismos:
+
+* Canal seguro a través de TLSv1.2 para la comunicación del cliente con Webpay.
+* Autenticación y autorización mediante el intercambio de headers `Tbk-Api-Key-Id` y `Tbk-Api-Key-Secret`.
+  
+#### Ambiente de Producción
+
+Las URLs de endpoints de producción están alojados dentro de
+<https://www.pagoautomaticocontarjetas.cl//>.
+
+```java
+// Host: PAGO CON  TARJETAS:CL
```

Comment:
👀

Reply:
No, I just copied them over and didn't check, it didn't seem important at the moment since we wanted to take this out fast